werkzeug/test.py

```diff
@@ -240,6 +240,8 @@ class EnvironBuilder(object):
             with the key and the unpacked `tuple` items as positional
             arguments.
         -   a `str`:  The string is set as form data for the associated key.
+    -   a file-like object: The object content is loaded in memory and then
```

Comment:
The three bullets use 4 space indentation and the rendering looks good to me.
![doc](https://cloud.githubusercontent.com/assets/49379/19001216/3b90b62c-8714-11e6-9a89-57c852ac175e.png)